CI: two tests fail with "ImportError: Install s3fs to access S3"

In the CI builds, the following test failures occur:

```
FAILED tests/test_writer.py::DatasetWriterTest::test_aws_s3_with_unknown_bucket - ImportError: Install s3fs to access S3
FAILED tests/test_writer.py::DatasetWriterTest::test_object_storage_params - ImportError: Install s3fs to access S3
```

See e.g. <https://ci.appveyor.com/project/bcdev/nc2zarr/builds/41635265/job/sxxbuetiy680cwc1>. s3fs is (apparently) installed and I can't reproduce locally, so seems to be an error in the CI environment or configuration. Maybe a recently introduced bug in micromamba?

--- a/environment.yml
+++ b/environment.yml
@@ -14,7 +14,7 @@ dependencies:
   - pandas
   - pyyaml
   - retry
-  - s3fs
+  - s3fs >=2022.1.0
   - xarray
   - zarr
   # Testing

